scripts/run_integration_tests_aarch64.sh

```diff
@@ -258,6 +258,14 @@ else
     exit $RES
 fi
 
+# Run tests on dbus_api
+if [ $RES -eq 0 ]; then
+    cargo build --features "dbus_api" --all --release --target $BUILD_TARGET
```

Comment:
Unfortunately the placement here means that all tests after this point (acpi, live migration, etc) will be with that dbus_api binary. You probably should put this at the end of the file.

Reply:
Right. Good catch! Fixing it.